cli/index.js

```diff
@@ -6,31 +6,82 @@ const { hideBin } = require('yargs/helpers');
 const { mochify } = require('@mochify/mochify');
 
 const opts = yargs(hideBin(process.argv))
+  .usage(
+    '$0 [options] <spec...>',
+    'Run Mocha tests using Mochify.',
+    (local) => {
+      local
+        .example(
+          '$0 --driver puppeteer --bundle browserify "./src/**/*.test.js" ',
+          'Bundle all files matching the given spec using browserify and run them using @mochify/driver-puppeteer.'
+        )
+        .example(
+          '$0 --config package.json "./src/**/*.test.js" ',
+          'Run all tests matching the given spec using the configuration from package.json\'s "mochify" key.'
+        )
+        .example(
+          '$0 --esm --reporter dot "./src/**/*.test.js" ',
+          'Run all tests matching the given spec as ES modules and use the "dot" reporter for output.'
+        );
+    }
+  )
   .option('config', {
-    type: 'string'
+    alias: 'C',
+    type: 'string',
+    group: 'Options:',
+    describe:
+      'The config file to use. In case `package.json` is given, the configuration is expected to be stored in a top-level "mochify" key. In case an option is present in both the config file and as a CLI flag, the flag takes precedence. Refer to the documentation of `@mochify/mochify` for available configuration options.'
```

Comment:
Any opinions on handling long strings like this in regards to line length?

Reply:
What happens if you use a multiline string? Does it screw up the layout?

Otherwise we can just concatenate strings with `+`.